Gemfile

```diff
@@ -38,7 +38,7 @@ gem 'rest-client', '>= 1.8.0'
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'manageiq-api-common', :git => 'https://github.com/ManageIQ/manageiq-api-common', :branch => 'master'
-gem 'topological_inventory-api-client', :git => "https://github.com/ManageIQ/topological_inventory-api-client-ruby", :branch => "master"
+gem 'topological_inventory-api-client', '~> 1.0', :git => "https://github.com/ManageIQ/topological_inventory-api-client-ruby", :branch => "master"
```

Comment:
@mkanoor the `:branch` argument is no longer needed when you declare the version attribute.

Reply:
@syncrou I think `:branch` is not required because `master` is the default.  As stated in https://bundler.io/gemfile.html#gemfiles

But I am not against explicitly setting it here.